packages/yarnpkg-libzip/artifacts/build.sh

```diff
@@ -98,18 +98,18 @@ build() {
     -s ENVIRONMENT=node \
     -s NODERAWFS=1 \
     -s SINGLE_FILE=1 \
+    -s NODEJS_CATCH_EXIT=0 \
+    -s NODEJS_CATCH_REJECTION=0 \
     "$@" \
     -I./libzip-"$LIBZIP_VERSION"/build/local/include \
     -O3 \
-    --llvm-lto 1 \
```

Comment:
This emits a warning that it's ignored

Reply:
From this thread it should have an effect, but mostly unnoticeable except added size - fine by me to remove it (does it decrease the bundle size? 🤔): https://github.com/emscripten-core/emscripten/issues/10603

Reply:
No size changes from what I can see, curious